controllers/front/SmailyRssFeed.php

```diff
@@ -32,7 +32,33 @@ public function initContent()
 
     public function generateRssFeed()
     {
-        $products = Product::getProducts($this->context->language->id, 0, 50, 'date_upd', 'desc', false, true);
+        // Hardcoded values in v1.3.0 and older.
+        $limit = 50;
+        $order_by = 'date_upd';
+        $order_way = 'desc';
+        $id_category = false;
```

Comment:
Rename `$id_category` to `$category_id` here.

Reply:
This code portion was obsolete put using` $category_id `variable on line 45